tests/integration/cli/test_export_inference.py

```diff
@@ -20,8 +20,8 @@ def _check_relative_metric_diff(ref: float, value: float, eps: float) -> None:
 
     avg = max(0.5 * (ref + value), 1e-9)
     diff = abs(value - ref)
-
-    assert diff / avg <= eps, f"Relative difference exceeded {eps} threshold. Absolute difference: {diff}"
+    is_positive_changed = value > ref or diff / avg <= eps
```

Comment:
@sovrasov i also fixed it to be true if there is a performance improvement after export.

Reply:
Ok, anyway it looks weird we still can't obtain deterministic results even for classification.

I've seen a PR that tests ocv vs ov resize, so there's a hope we won't have such discrepancy starting from the next OV version.